docker/Dockerfile

```diff
@@ -21,15 +21,15 @@ RUN cargo build --locked --profile $PROFILE $BUILD_ARGS
 # ==========================
 # stage 2: packaging
 # ==========================
-FROM ubuntu:22.04
+FROM ubuntu:20.04
 LABEL maintainer="Trust Computing GmbH <info@litentry.com>"
 
 ARG PROFILE
 
 COPY --from=builder /litentry/target/$PROFILE/litentry-collator /usr/local/bin
-
+RUN echo "deb http://th.archive.ubuntu.com/ubuntu jammy main" >> /etc/apt/sources.list
```

Comment:
jammy is 22 while it's operating on 20. I'm not sure if it's a good idea to mix repos from various versions.